src/mono/mono/metadata/object.c

```diff
@@ -1518,6 +1518,8 @@ build_imt_slots (MonoClass *klass, MonoVTable *vt, gpointer* imt, GSList *extra_
 				 * add_imt_builder_entry anyway.
 				 */
 				method = mono_class_get_method_by_index (mono_class_get_generic_class (iface)->container_class, method_slot_in_interface);
+				if (m_method_is_static (method))
+					continue;				
```

Comment:
We're going to need to revisit this (and line 1535, below) when non-abstract static virtual methods are going to be supported